packages/beacon-node/src/api/impl/beacon/blocks/utils.ts

```diff
@@ -65,6 +65,14 @@ async function resolveBlockIdOrNull(
     };
   }
 
+  if (blockId === "justified") {
+    const justified = forkChoice.getJustifiedBlock();
+    return {
+      block: await db.block.get(fromHexString(justified.blockRoot)),
+      executionOptimistic: false,
```

Comment:
You can compute the correct executionOptimistic boolean value with the forkChoice returned data structure. Check other API endpoints can't recall the exact helper fn name